src/main/java/domain/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,88 @@
+package domain.lotto;
+
+import domain.result.LottoRank;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    private static final int LOTTO_NUMBERS_SIZE = 6;
+    private static final int SPLIT_THRESHOLD = -1;
+    private static final String DELIMITER = ", ";
+
+    private final List<LottoNumber> lottoNumbers;
+
+    private LottoNumbers(final List<LottoNumber> lottoNumbers) {
+        List<LottoNumber> copy = new ArrayList<>(lottoNumbers);
+        validateLottoNumbers(copy);
+        this.lottoNumbers = copy;
+    }
+
+    public LottoNumbers(final String lottoNumbers) {
+        String[] temp = lottoNumbers.split(DELIMITER, SPLIT_THRESHOLD);
+        List<LottoNumber> tempLottoNumbers = new ArrayList<>();
+        for (int i = 0; i < temp.length; i++) {
+            tempLottoNumbers.add(LottoNumber.of(temp[i]));
+        }
+        validateLottoNumbers(tempLottoNumbers);
+        this.lottoNumbers = tempLottoNumbers;
```

Comment:
```suggestion
        this(tempLottoNumbers);
```

인스턴스 변수의 초기화는 하나의 생성자에서 해주는 것이 좋습니다.
초기화 또한 중복이라는 관점에서 어떠한 문제가 있을지 고민해보아요!

Reply:
네 공부해보겠습니다!